### PR TITLE
Reimplement empty state below editor

### PR DESCRIPTION
**Pull request by Yogeshp0012**
**Description**

- Added the girlwithlaptop icon above the nothing to show message.
- Added an additional text.

image

**Related issue(s)**

 Resolves [#6](https://github.com/janvi01/sql-editor/issues/6)

### CODE REVIEW

File: src/components/Output/OutputDisplay.js [hunk 1]
```diff
@@ -60,9 +61,23 @@ const OutputDisplay = ({ submittedQuery, loading, setLoading }) => {
...
+            </Center>
+            <Center>
+                <Box textAlign="center">
+                    <Heading as="h1" fontSize="xl" mt={4}>
+                        Nothing to show at the moment
+                    </Heading>
+                    <Text mt={2}>Run a query first to see resulting tablet</Text>
+                </Box>
+            </Center>
+        </Box>
```
Comment (janvi01) [inline src/components/Output/OutputDisplay.js:80]:
```suggestion
        <VStack justifyContent={"center"} p={4}>
          <Image
            src={girlWithLaptopImage}
            alt="Girl With Laptop"
            maxH="300px"
          />
          <Heading as="h1" fontSize="xl" mt={4}>
            Nothing to show at the moment
          </Heading>
          <Text>Run a query first to see resulting tablet</Text>
        </VStack>
```

**Review by janvi01 — COMMENTED** [review]
Hey @Yogeshp0012 Thanks for the PR. The code and the preview LGTM. Well done!

It can be optimized too instead of using various components, we can use a vertical stack simply with justified center content here.

Please make the required changes. Also while creating a PR, always create a new branch. Keep this in mind from the next time (not needed now).

File: src/components/Output/OutputDisplay.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React, { useEffect, useState } from "react";
 import OutputTable from "./OutputTable";
-import { Button, HStack, Heading, Spacer, Spinner } from "@chakra-ui/react";
+import { Box, Button, Center, HStack, Heading, Image, Spacer, Spinner, Text } from "@chakra-ui/react";
```
Comment (janvi01) [inline src/components/Output/OutputDisplay.js:3]:
```suggestion
import { Button, HStack, Heading, Image, Spacer, Spinner, Text } from "@chakra-ui/react";
```

**Review by janvi01 — CHANGES_REQUESTED** [review]
make sure to format the file also.

**Comment by Yogeshp0012** [comment]
Done @janvi01.

**Comment by janvi01** [comment]
LGTM 🚀 @Yogeshp0012

**Comment by Yogeshp0012** [comment]
Hi @janvi01 also can you add the hacktoberfest-accepted  label to make sure this comes under hacktoberfest.

**Pull request merged**